app/controllers/items_controller.rb

```diff
@@ -28,16 +29,16 @@ def edit
 
   def update
     if authorized_user(@user) && @item.update(item_params)
-      flash[:notice] = "Item, #{@item.name}, updated."
+      flash[:notice] = "#{@item.name}, updated."
       redirect_to group_list_item_path(@group, @list, @item)
     else
-      flash[:warning] = 'An error occurred, please try again.'
+      flash[:warning] = @item.errors.full_messages.to_sentence
```

Comment:
There is an opportunity to make this more DRY by pulling out a method that does this `model.errors.full_messages.to_sentence`. Not a high-priority item but could be useful to put in a helper (not helper method but Rails helper class). Not a required fix for this PR imo